saleor/discount/migrations/0046_move_codes_to_new_model.py

```diff
@@ -0,0 +1,34 @@
+# Generated by Django 3.2.20 on 2023-08-21 07:23
+
+from django.db import migrations
+
+
+def move_codes_to_new_model(apps, schema_editor):
+    Voucher = apps.get_model("discount", "Voucher")
+    VoucherCode = apps.get_model("discount", "VoucherCode")
+
+    voucher_codes = []
+    for voucher in Voucher.objects.values("id", "code", "used", "usage_limit"):
```

Comment:
Question is how many vouchers users can have, I wonder if it should be done in batches or not. 
Zero-downtime support should be also taken into consideration

Reply:
yes, I was planning to add all zero-downtime migrations in separate PR

Reply:
But we still should consider if this update shouldn't be made in batches. Creating all VoucherCodes in one query might fail.